app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -1,78 +1,168 @@
-            </div>
-            <div>
-              <AuHeading @level="2" @skin="4">
-                {{@model.roadSignConceptCode}} -
-                {{#each @model.categories as |category|}}
-                  <span>
-                    {{category.label}}
-                  </span>
-                {{/each}}
-              </AuHeading>
-              <p>
-                {{@model.meaning}}
-              </p>
-            </div>
-          </div>
-        </c.header>
-        <c.content />
-        <c.footer>
-          <LinkTo
-            class="au-c-button"
-            @route="road-sign-concepts.edit"
-            @model={{@model.id}}
-          >
-            {{t "roadSignConcept.crud.edit"}}
-          </LinkTo>
+<AuToolbar @size="large">
+  <AuToolbarGroup>
+    <AuHeading @skin="2">
+      {{t "roadSignConcept.details"}}
+    </AuHeading>
+  </AuToolbarGroup>
+</AuToolbar>
+<AuToolbar @size="large">
+  <AuToolbarGroup>
+    <ul class="au-o-grid">
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <img alt="" role="presentation" src={{@model.image}} />
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.code"}}
+        </AuLabel>
+        <p>
+          {{@model.roadSignConceptCode}}
+        </p>
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-6@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.categories"}}
+        </AuLabel>
+        {{#each @model.categories as |category|}}
+        <p>
+          {{category.label}}
+        </p>
+        {{/each}}
+      </li>
+      <li class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
+        <AuLabel>
+          {{t "roadSignConcept.attr.meaning"}}
+        </AuLabel>
+        <p>
+          {{@model.meaning}}
+        </p>
+      </li>
+    </ul>
+  </AuToolbarGroup>
+</AuToolbar>
 
-          <AuButton @alert={{true}} {{on "click" (fn (mut this.isOpen) true)}}>
-            {{t "roadSignConcept.crud.delete"}}
-          </AuButton>
-          <AuModal
-            @modalTitle={{t "utility.confirmation.title"}}
-            @modalOpen={{this.isOpen}}
-            @closeModal={{fn (mut this.isOpen) false}} as |Modal|
-          >
-            <Modal.Body>
-              <p>
-                {{t "utility.confirmation.body"}}
-              </p>
-            </Modal.Body>
-            <Modal.Footer>
-              <AuButton
-                @alert={{true}}
-                {{on "click" (fn this.removeRoadSignConcept @model)}}
-              >
-                {{t "roadSignConcept.crud.delete"}}
-              </AuButton>
-            </Modal.Footer>
-          </AuModal>
-        </c.footer>
-      </AuCard>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+<AuDataTable @content={{this.model.subSigns}} @fields="Image Meaning Code"
+  @noDataMessage={{t "roadSignConcept.crud.noData" }} @size={{this.size}} @page={{this.page}} @sort={{this.sort}} as
```

Comment:
The `this.size`, `this.page` and `this.sort` values do not exist, so this passes `undefined` to the component. I think it might be better to not use the AuDataTable component, and use add the needed html structure yourself. This component is only useful if you want server side paginated tables. Another (more hacky) option would be to just pass in hardcoded values (1, 1 and ""). I think that should fix the NaN's in the pagination as well.